### PR TITLE
Update Booklooker.de.xml, fix #15866

### LINKED ISSUE #15866: Wrong host in Booklooker.de.xml
Type: ruleset issue

From https://trac.torproject.org/projects/tor/ticket/25583

> booklooker.de does not use the host ​https://secure.booklooker.de anymore.
> All SSL connections use ​https://www.booklooker.de

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/Booklooker.de.xml [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
+		- bjoerk.booklooker.de
+-->
 <ruleset name="booklooker.de">
-  <target host="booklooker.de" />
-  <target host="*.booklooker.de" />
+	<target host="booklooker.de" />
+	<target host="www.booklooker.de" />
+	<target host="api.booklooker.de" />
+	<target host="bms.booklooker.de" />
+	<target host="dev.booklooker.de" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Booklooker.de.xml]: This target has a different behaviour over HTTP and HTTPS. Plus, we don't have a working test url. Given we had issues recently with securing `dev` subdomains, I'd rather not add it.

File: src/chrome/content/rules/Booklooker.de.xml [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
+	<target host="download.booklooker.de" />
+	<target host="falco.booklooker.de" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Booklooker.de.xml]: Refused over HTTP and 403 over HTTPS. I'm not sure we should add it.

File: src/chrome/content/rules/Booklooker.de.xml [hunk 3]
```diff
@@ -1,9 +1,20 @@
...
+-->
 <ruleset name="booklooker.de">
-  <target host="booklooker.de" />
-  <target host="*.booklooker.de" />
+	<target host="booklooker.de" />
+	<target host="www.booklooker.de" />
+	<target host="api.booklooker.de" />
+	<target host="bms.booklooker.de" />
+	<target host="dev.booklooker.de" />
+	<target host="download.booklooker.de" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Booklooker.de.xml]: I would be good to have a test url here

File: src/chrome/content/rules/Booklooker.de.xml [hunk 4]
```diff
@@ -1,9 +1,20 @@
...
+	<target host="falco.booklooker.de" />
+	<target host="images.booklooker.de" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Booklooker.de.xml]: Please add a test url such as https://images.booklooker.de/m/9783833310539/Jenny-Colgan+Die-kleine-B%C3%A4ckerei-am-Strandweg.jpg

File: src/chrome/content/rules/Booklooker.de.xml [hunk 5]
```diff
@@ -1,9 +1,20 @@
...
+	<target host="images2.booklooker.de" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Booklooker.de.xml]: Same as `falco` and `dev`. I'm open to discussion but I'm inclined to remove it.

**Comment by cschanaj** [comment]
@Bisaloo I have removed the hosts with inconsistent behavior over HTTP and HTTPS, but I cannot find a test url for `download` from a quick google search. I shall remove that as well depending on your opinion. thanks!

**Comment by Bisaloo** [comment]
There is a 301 redirect to download so let's keep it.

**Comment by Bisaloo** [comment]
stats should be removed as well

**Pull request merged**